genie-test/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,34 @@
+<!--
+/*
+ *
+ *  Copyright 2015 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+ -->
+ <configuration>
```

Comment:
Was something wrong with how boot was doing it by default? Can't we change the logging levels via the standard spring properties `logging.level.{package}`

Reply:
Oh, didn't think of that. 
However this applies more broadly to **all** tests.
Netty and gRPC are extremely verbose so i wanted to shut them down.